packages/frontend/src/hooks/pivotTable/pivotQueryResults.ts

```diff
@@ -56,10 +56,18 @@ export const pivotQueryResults = ({
     if (rows.length === 0) {
         throw new Error('Cannot pivot results with no rows');
     }
-    const columnOrder = pivotConfig.columnOrder || [];
+
+    const hiddenFieldIds = pivotConfig.hiddenFieldIds || [];
+
+    const columnOrder = (pivotConfig.columnOrder || []).filter((id) => {
+        return !hiddenFieldIds.includes(id);
+    });
+
     // Headers (column index)
     const headerDimensions = pivotConfig.pivotDimensions.filter(
```

Comment:
hiding dimensions, will mean that data gets lost in the pivot, depending on the order of the data, the last value in the group of selected dimensions will be used and all the others will be discarded